src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -214,6 +214,18 @@ export class ResultsViewPageStore {
         //     this.ajaxErrors.push(error);
         // });
         this.getURL();
+
```

Comment:
this seems super icky to me.  this is so we don't make further calls to it?

Reply:
thats so the oncoprint setting goes to false instead of the default true. at this point it will become impossible to change from the UI. maybe it would be a better idea to model this situation more explicitly in mutationAnnotationSettings